Buildings/Controls/OBC/ASHRAE/G36_PR1/TerminalUnits/Controller.mo

```diff
@@ -5,10 +5,7 @@ block Controller "Controller for room VAV box"
     final unit="s",
     final quantity="Time")
     "Sample period of component, set to the same value as the trim and respond that process yPreSetReq";
-  parameter Real V_flow_nominal(
```

Comment:
Removing the parameter `V_flow_nominal` is a non-backward compatible change. This needs to be listed in the release notes of the top-level `package.mo` file under the non-backward compatible changes, and an entry must be made in the conversion script.